fastai/data/external.py

```diff
@@ -110,15 +112,24 @@ class URLs():
     WT103_FWD          = f'{S3_MODEL}wt103-fwd.tgz'
     WT103_BWD          = f'{S3_MODEL}wt103-bwd.tgz'
 
-    def path(url='.', c_key='archive'):
+    def path(
+        url='.', # URL of file to download
+        c_key='archive' # Name of a folder in `fastai_cfg` to save the file. Should be one of [`'data', 'archive', 'storage', 'model'`]
```

Comment:
I think "Key in `Config` where to save URL" would be better

Reply:
Done